libcontainer/init_linux.go

```diff
@@ -151,14 +151,14 @@ func finalizeNamespace(config *initConfig) error {
 	if err := system.ClearKeepCaps(); err != nil {
 		return errors.Wrap(err, "clear keep caps")
 	}
-	if err := w.ApplyCaps(); err != nil {
-		return errors.Wrap(err, "apply caps")
-	}
 	if config.Cwd != "" {
```

Comment:
You could move this before all of the capability and `setupUser` code (after `CloseExecFrom`).

Reply:
okay